frontend/test/metabase-lib/lib/Question.unit.spec.js

```diff
@@ -102,6 +109,136 @@ const orders_count_by_id_card = {
   },
 };
 
+const people_count_by_state_card = {
+  id: 2,
+  name: "# people by state",
+  display: "table",
+  visualization_settings: {},
+  dataset_query: {
+    type: "query",
+    database: SAMPLE_DATABASE.id,
+    query: {
+      "source-table": PEOPLE.id,
+      breakout: [["field", PEOPLE.STATE.id, null]],
```

Comment:
nit: you can use `Field.instance.reference` to get MBQL field reference to make it less verbose. Although some tests might actually benefit from explicitly defined queries. Anyway, it works like that:

```js
PEOPLE.STATE.reference()
// would return
[ "field", PEOPLE.STATE.ID, null ]
```